app/redux/reducers/users.js

```diff
@@ -0,0 +1,46 @@
+const SET_SELECTED_USER = 'SET_SELECTED_USER';
```

Comment:
Can you check the groups.js reducer and copy the formatting for keeping things neat?  with the horizontal comments and the order of the page matching groups.